src/configure/configurers/localGithubWorkflowConfigurer.ts

```diff
@@ -258,8 +258,6 @@ export class LocalGitHubWorkflowConfigurer implements Configurer {
                         metadata["properties"]["configName"] = `${doc["name"]}`;
                     }
                     metadata["properties"]["configPath"] = `${configPath}`;
-                    metadata["properties"]["repoUrl"] = `https://github.com/${inputs.sourceRepository.repositoryId}`;
```

Comment:
This is in case of local configurer? If so then where will these be set?

Reply:
It is being set as part of source control api. This was added to make it consistent to provisioning service but this results in a double repo url because of some bug in source control api. In provisioning service we are updating first metadata and scmtype and then the sourcecontrol so we are not facing this issue for deployment configured using provisioning service. But here we are doing, first we are doing source control update and then metadata update which is causing this issue.

Reply:
Should we be consistent here in both local and remote configurer. we can use the same order which provisioning svc is using.

Reply:
After changing the order, it works fine. The error is happening because of repoUrl key case mismatch for different web apps.